src/github.org

```diff
@@ -0,0 +1,99 @@
+#+TITLE: GitHub Training 2017
+#+AUTHOR: Sahit Chintalapudi, Josh Ting 
+#+EMAIL: schintalapudi@gatech.edu, josh.ting@gatech.edu
+#+OPTIONS: toc:nil timestamp:nil reveal_control:t num:nil reveal_history:t tags:nil author:nil
+
+* What is Git?
+- Git is a "Distributed Version Control System" (DCVS)
+- It is the tool RoboJackets uses to collaborate on code
+- Git tracks changes made to files in your project, and handles merging
+  these changes
+- Working on different parts of code and then combining them creates unique challenges that
+  cannot be handled by file sharing systems like DropBox and Google Drive
+
+* What is GitHub?
+- GitHub is a publically hosted git server
+- It hosts our projects as "repositories" on the internet
+- This saves us the hassle of hosting a server for fellow RoboJackets to
+  obtain copies of the code
+- As you matriculate through Tech, you'll be building an online portfolio of
+  your work on GitHub.
+
+* How do you find things on GitHub?
+- Repositories are stored at https://github.com/<ORG-OR-USER-NAME>/<REPO-NAME>
+- So all of our projects can be found at https://github.com/RoboJackets
+- Your personal projects will exist at https://github.com/<YOUR-GH-USERNAME>
+
+** Create a GitHub account
+- [[https://github.com/join%0A][https://github.com/join]]
+- Please include /at least/ your real first name
+# Emphasize this requirement is for display name on GitHub, not username
+
+[[file:https://i.imgur.com/0cdXQXW.png]]
+
+* Fork and star our repo
+- Navigate to github.com/robojackets/robocup-software
+    - Or the repo of whatever RoboJackets team you are on
+
+[[file:https://i.imgur.com/kYzz2oh.png]]
+
+* What is a fork?
+- Problem: How do I try something out on a project without affecting the project itself?
+- Solution: Forks!
+- A fork is a copy of a repository that exists on your personal account
+- Changes made on a fork will not affect the original repository, and vice versa
+- Your fork is present on https://github.com/<YOUR-GH-USERNAME>/robocup-software
+
+* Install Git on your computer
+- Windows: https://git-scm.com/download/win
+- Mac OS: https://git-scm.com/download/mac
+- Linux: https://git-scm.com/download/linux
+
+* Git Config
+- These commands set up git to attribute your contributions to our code to you.
+
+  #+BEGIN_SRC shell
+    git config --global user.name "Your Name"
```

Comment:
This is not really your problem, but could you change this: https://github.com/RoboJackets/software-training/blob/master/rj-software-training.el#L85 to a `nil` instead of a `t`. Right now, I overrode the org mode defaults (from my personal config) to prevent stripping indentation (because python is really annoying), but it's causing these blocks to be indented 2 spaces in. The defaults should work the best in this project though.

I'll do it after this PR if you don't get around to it.